python/lsst/pipe/base/pipelineIR.py

```diff
@@ -368,12 +365,9 @@ def maybe_merge(self, other_config: "ConfigIR") -> Generator["ConfigIR", None, N
     def __eq__(self, other: object) -> bool:
         if not isinstance(other, ConfigIR):
             return False
-        elif all(
-            getattr(self, attr) == getattr(other, attr) for attr in ("python", "dataId", "file", "rest")
-        ):
-            return True
-        else:
-            return False
+        return bool(
+            all(getattr(self, attr) == getattr(other, attr) for attr in ("python", "dataId", "file", "rest"))
+        )
```

Comment:
`all` is already `bool`

Reply:
This is what I get from using automated tools... Will fix.